jumeg/data/lobes.yaml

```diff
@@ -0,0 +1,17 @@
+- occipital:
```

Comment:
You could rename this file to lobes_grouping.yaml so we know that it is a grouping file.

Reply:
Good idea.